search-service/src/main/kotlin/com/egm/stellio/search/service/EntityPayloadService.kt

```diff
@@ -328,9 +329,25 @@ class EntityPayloadService(
                     postfix = ")"
                 ) { "'$it'" }
             else null
+        val formattedDatasetId =
```

Comment:
I don't get why you added modifications for `queryEntity` and `queryEntities`, the `datasetId` projection parameter was already handled for these 2 operations.